addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -1,9 +1,20 @@
 @include media-breakpoint-down(md) {
-    img, .media_iframe_video, span.fa, i.fa {
+    img, .media_iframe_video {
         &:not(.o_animate) {
             transform: none !important;
         }
     }
+
+    // For RTL (Right-to-Left) purposes, don't disable the transform directive
+    // for Font Awesome icons that are flipped by 180deg.
+    $not-selectors: '';
+    $rtl-fa-classes: () !default;
```

Comment:
@rdeodoo Without this line the "sign" module fails to load the assets because $rtl-fa-classes is not loaded. "fontawesome_overridden.scss" is not anymore in the manifest in 16.4. I don't know why. So if it's not necessary, I suppose that I can declare this variable empty by default. Do you want a fix on enterprise?

Reply:
@rdeodoo https://github.com/odoo/enterprise/commit/5fa63a2f284fe93acc0c4d8dc12ee47646703247

Reply:
It means that `/web/static/src/legacy/scss/fontawesome_overridden.scss` is not in the sign asset somehow somewhere, right?
Then you reach this file which is crashing because it was not loaded, and thus not the variable either.
Wouldn't the fix be about adding that file instead?

Reply:
@rdeodoo Exactly. So I'll create a fix on enterprise sign module for this PR.